src/parser/csql_grammar.y

```diff
@@ -1100,6 +1101,7 @@ static char *g_plcsql_text;
 %token ATTACH
 %token ATTRIBUTE
 %token AVG
+%token AUTHID
```

Comment:
Do I need to register as a keyword?
If so, please refer to the keywords[] variable in the keyword.c file.
If it can be used as an identifier, please refer to the identifier definition in csql_grammar.y.

Please also review the tokens added below.

Reply:
As Oracle, they don't have to be reserved as an identifier. Thank you for the review!